Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/view_routes.py

```diff
@@ -169,5 +173,6 @@ def constants(self):
         return Response(
             self.environment.get_template('constants.js').render(
                 XcodeCloud=self.suite_types.get('XcodeCloud') or [],
+                default_architecture=json.dumps(self.default_architecture) if self.default_architecture else 'null',
```

Comment:
@JonWBedard I know this has landed now, but we don't actually need the `else 'null'` case here to handle `None`, `json.dumps` will do that fine